### ISSUE TITLE
Add dependency to bootstrapPlatform for tasks created by HybrisAntRule

### ISSUE THREAD

**Issue by guiliguili** [SAP/commerce-gradle-plugin#24]
As a developper I would like to :
* Use a unique tool to setup, build and run the platform : Gradle 
* Minimise the number of tasks to execute manually to be able to build and run.
* Use the [latest available SAP Commerce patch version](https://help.sap.com/viewer/1be46286b36a4aa48205be5a96240672/v2011/en-US/811b9e1cb1094da5bbe8e384345e73cc.html) in the Maven repository.

Currently we must run the `bootstrapPlatform` before running `ybuild` or any other task created by the 
the [Task Rule](https://github.com/SAP/commerce-gradle-plugin/blob/master/docs/Plugin-sap.commerce.build.md#task-rules).

Suggestion : Add dependency to `bootstrapPlatform` for all tasks created by `HybrisAntRule`. Potentially this could be drive by inclusion/exclusion configuration properties with sensitive default values (include all, exclude none).

**Comment by mpern** [comment]
Interesting idea!

I'm thinking about adding a configuration option to configure the default dependencies for all tasks generated by the rule, something like:

```kts
hybris {
    antTaskDependencies.set(listOf("bootstrapPlatform"))
}
```

### Pros:
- doesn't break existing behavior
- task dependency is not hardcoded and can be adapted to project-specific setup

### Cons
- yet another configuration option

What do you think?

**Comment by mpern** [comment]
In the mean time, you can always configure regular [Gradle task dependencies][deps] for all relevant ant tasks by hand, e.g.

```gradle
tasks.named("yall").configure {
    dependsOn("bootstrapPlatform")
}
``` 

[deps]: https://docs.gradle.org/current/userguide/tutorial_using_tasks.html#sec:task_dependencies

**Comment by mpern** [comment]
Configuration option released with [v3.2.0]

[v3.2.0]:https://github.com/SAP/commerce-gradle-plugin/releases/tag/v3.2.0
